### DISCUSSION TITLE
For those with printers that can only print up to 200mm x 200mm

### DISCUSSION THREAD

**Discussion by ziti** [martinwoodward/octolamp#13] [Q&A] [answered]
This is a great project and one that I happen to already have the parts for. :) For those with printers that can only print up to 200mm x 200mm would this design still work if scaling is used before slicing to get the size down? If I set the scaling in Cura to 95%, the model is 196mm x 196mm.

Thanks!

**Comment by ziti** [comment]
As an experiment, I printed the 3 parts at 25% scale and they all fit together perfectly. Looks like it will work! I'll report back after I print the parts at 90%.

**Comment by ziti** [comment, marked as answer]
Scaling to 90% worked perfectly!

**Reply by martinwoodward** [reply]
That's great to hear thanks @ziti - the cable stuff at the back might be a tight fit but hopefully left enough room elsewhere.  You all soldered up and things still working ok?

**Reply by ziti** [reply]
Yes, I have the lights (89 of em) in the inner body tested them with the D1 and everything worked. Just waiting for the reflector to finish printing since I accidently turned off the printer when I went to plug in my soldering iron. So far everything seems to fit! I'll post a picture once it is complete.

Thanks for this fun project!

**Reply by ziti** [reply]
It was a tight fit, but I got everything together!
